cli/cmd/vuln_container.go

```diff
@@ -61,6 +66,9 @@ Arguments:
 		Long:    "Check the status of an on-demand container vulnerability assessment.",
 		Args:    cobra.ExactArgs(1),
 		RunE: func(_ *cobra.Command, args []string) error {
+			if err := validateFailureFlags(); err != nil {
```

Comment:
Need to look into handling validation better. Not a fan of pasting this into all the commands the have this flag enabled.